src/api/app/assets/javascripts/webui/comment.js

```diff
@@ -14,7 +14,22 @@ function updateCommentCounter(selector, count) {
   $(selector).text(parseInt(oldValue) + count);
 }
 
+function validateForm(e) {
+  var submitButton = $(e.target).closest('[class*="-comment-form"]').find('input[type="submit"]');
+  if ($(e.target).val().length > 0) {
```

Comment:
Could this `if {} else {}` be changed to this:

```suggestion
    submitButton.prop('disabled', !$(e.target).val());
```

Reply:
Thank you! This looks really nice. I applied the suggested change:)